src/nft-card.ts

```diff
@@ -80,7 +80,8 @@ export class NftCard extends LitElement {
       }
       .card {
         background-color: transparent;
-        font-family: Avenir Next, Avenir, Helvetica Neue, sans-serif;
+        font-family: 'Roboto', sans-serif;
```

Comment:
where else is Roboto being used? we're not `<link>`ing to the googlefonts link for it right? I don't think all operating systems ship with it

Reply:
Yeah it is imported [here](http://github.com/ProjectOpenSea/embeddable_nfts/blob/0e3ef2516879ada8f02bbf916cbea9a9bbd252a4/src/nft-card.ts#L231-L231)